source/educators/references/advanced_features/managing_teams_via_csv.rst

```diff
@@ -57,45 +57,44 @@ team-set in the header row. For example, here are 8 students assigned across dif
     garrett,         verified, ,                 Team C
     hannah,          masters,  Team A,           Team 1
 
-**Note** that since teams are only unique within a team-set, the “Team A” that hannah is a member of for the
-“discussion-teams” team-set is different from the “Team A” in “case-studies” that alice and derek are assigned to.
 
+.. note:: Since teams are only unique within a team-set, the “Team A” that hannah is a member of for the “discussion-teams” team-set is different from the “Team A” in “case-studies” that Alice and Derek are assigned to.
 
 Managing Teams
----------------
+**************
 
-.. note:: Prerequisites: at least one managed team-set configured for course
+.. note:: Prerequisites: at least one managed team-set configured for course.
 
 From the **Teams > Manage** page, an instructor can use a CSV file, formatted per the schema above, to manage team memberships.
 
 An instructor should first get current memberships by going to the **View Current Team Memberships** section and
-clicking the *Download Memberships* button. This downloads a membership CSV file showing all enrolled students
+clicking the **Download Memberships** button. This downloads a membership CSV file showing all enrolled students
 and configured team-sets for the course.
 
 Staff can then modify this document (as below) before browsing to the file and uploading it in the
 **Assign Team Memberships** section.
 
 
 Actions
-~~~~~~~~
+=======
 
 A **user can be added or moved** to a ``team`` by adding the ``team-name`` in the appropriate ``team-set`` column.
 If the team does not already exist within the team-set a **new team will be created**.
 
-.. note:: ``team-name`` and ``team-set`` * **are case sensitive.** *
+.. note:: ``team-name`` and ``team-set`` * are case sensitive.
```

Comment:
The bolding has been removed but now we've got a stray asterisk in here. I recommend leaving this bolded and removing the extra asterisk.